<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>TableView</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>PlainTextEdit</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelPath">
         <property name="text">
          <string>当前文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurCell">
         <property name="text">
          <string>当前单元格：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCellText">
         <property name="text">
          <string>单元格内容：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actModelData"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlignRight"/>
   <addaction name="actFontBold"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/open.bmp</normaloff>:/images/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/save.bmp</normaloff>:/images/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>另存文件</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/append.bmp</normaloff>:/images/icons/append.bmp</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/insert.bmp</normaloff>:/images/icons/insert.bmp</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/delete.bmp</normaloff>:/images/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/import1.bmp</normaloff>:/images/icons/import1.bmp</iconset>
   </property>
   <property name="text">
    <string>模型数据预览</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/508.bmp</normaloff>:/images/icons/508.bmp</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/510.bmp</normaloff>:/images/icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/512.bmp</normaloff>:/images/icons/512.bmp</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/500.bmp</normaloff>:/images/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
